Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1

```diff
@@ -132,29 +137,39 @@ if (-not $sts) {
         }
         Write-LogInfo "File has been successfully copied to guest VM '${vmName}'"
 
-        $sts = Check-FileInLinuxGuest  -vmPassword $VMPassword -vmPort $VMPort -vmUserName $VMUserName -ipv4 $Ipv4   -fileName "/mnt/test/$testfile"
+        $sts = Check-FileInLinuxGuest  -vmPassword $VMPassword -vmPort $VMPort -vmUserName $VMUserName -ipv4 $Ipv4 -fileName "/mnt/test/$testfile" -checkSize $true
         if (-not $sts) {
             Write-LogErr "File check error on the guest VM '${vmName}'!"
             return "FAIL"
             break
         }
-        Write-LogInfo "The file copied matches the ${originalFileSize} size."
+        if (($sts/1gb) -eq ($fileSize/1gb) ) {
+            Write-LogInfo "The file copied matches the $fileSize size."
+        }
+        else {
+            Write-LogErr " The file copied doesn't match the $fileSize size!"
+            return "FAIL"
+            break
+        }
+
+        Write-LogInfo "The file copied matches the ${FileSize} size."
 
-        $sts = Run-LinuxCmd -username $VMUserName -password $VMPassword -port $VMPort -ip $Ipv4 "rm -f /mnt/test/$testfile" -runAsSudo
+        $sts =  Run-LinuxCmd -username $VMUserName -password $VMPassword -port $VMPort -ip $Ipv4 "rm -f /mnt/test/$testfile && echo 1 || echo 0 " -runAsSudo
         if (-not $sts) {
             Write-LogErr "Failed to remove file from VM $VMName."
             return  "FAIL"
             break
         }
         Write-LogInfo "File has been successfully removed from guest VM '${vmName}'"
-#
-# Removing the temporary test file
-#
-Remove-Item -Path \\$HvServer\$file_path_formatted -Force
-if (-not $?) {
-    Write-LogErr "Cannot remove the test file '${testfile}'!"
-    return "FAIL"
-}
+    }
+    #
+    # Removing the temporary test file
+    #
+    Remove-Item -Path \\$HvServer\$file_path_formatted -Force
+    if (-not $?) {
+        Write-LogErr "Cannot remove the test file '${testfile}'!"
+        return "FAIL"
```

Comment:
do we want to fail the test if the cleanup fails?
I mean if the fcopy functionality is fine and the failed cleanup will not affect new runs, then the LogErr should be enough.

Reply:
yes,  have remove "return ‘FAIL’" in all the scripts.